### ISSUE TITLE
Add HD Texture Support and Tomb1Main 

### ISSUE THREAD

**Issue by userfrendly123** [XProger/OpenLara#465]
Thanks for your Work!
Is it possible to add HD texture support from http://download.glidos.net/TombATI/TombATI_HD_Textures_v1.0.exe and https://www.tombraiderxtra.com/ here and also integration/support  from https://github.com/LostArtefacts/Tomb1Main?

**Comment by XProger** [comment]
no, I don't like it

**Comment by Kroc** [comment]
The ability to replace textures is still very useful, even if you don't like the HD texture packs. Consider the DOOM sprite fixup project that fixes a large number of minor mistakes in the original weapon sprites.

"The customer is always right _in matters of taste_".

Texture packs could be hideous to you, but the functionality still remains a useful tool that may lead to something else better, even essential, down the line.

**Comment by repoleveDevil** [comment]
This texture pack quoted by userfrendly123 is actually very good. I closed tomb1 through Openlara, I faced some bugs that I will report in other posts. But what really bothered me were the original textures that don't match the modern look of openlara. I'm playing again now through Tomb1Main and the hd textures really make a lot of difference, and morally? What's the problem with opening texture support? A simple response like "no, I don't like it" knocked out the credit I had on this project.
